Dockerfile

```diff
@@ -812,7 +815,7 @@ RUN curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh |
 # RUN apk add --no-cache dotnet9-sdk
 # Next line commented because already managed by another linter
 # ENV PATH="${PATH}:/root/.dotnet/tools"
-    && dotnet tool install --global TSQLLint
+    && dotnet tool install --global TSQLLint --version ${SQL_TSQLLINT_VERSION}
```

Comment:
When was quoting required already?

Reply:
I assume you mean it could simply be `--version $SQL_TSQLLINT_VERSION`?

Reply:
With double quotes. I remember a couple months ago I bumped myself into this when using args..

Reply:
Done!